actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -64,8 +77,9 @@ def output_type
             output_type
           end
 
-          def display_image
-            message = "[Screenshot]: #{image_path}\n".dup
+          def display_screenshot
+            message = "[Image screenshot]: file://#{image_path}\n".dup
+            message << "     [HTML screenshot]: file://#{html_path}\n"
```

Comment:
I'm not sure if `HTML screenshot` is the best name here. What about to use just `HTML` or `HTML output` or `rendered HTML`,... :question:

Reply:
Yeah, agreed. Maybe `page` is clear enough? It also matches `capybara` naming `save_page` vs `save_screenshot`.

Reply:
I pushed https://github.com/rails/rails/pull/31960/commits/0c202af31f31e5600eeec86e3ea1773bc9eb427a, what do you think?